pkg/sql/sem/tree/type_check.go

```diff
@@ -450,6 +450,10 @@ func resolveCast(
 		telemetry.Inc(GetCastCounter(fromFamily, toFamily))
 		return nil
 
+	case toFamily == types.BytesFamily && fromFamily == types.EnumFamily:
```

Comment:
i think we want to delete https://github.com/cockroachdb/cockroach/blob/d09d14178a72f7625d0869902b7087a3f1ee3415/pkg/sql/sem/tree/cast.go#L1350-L1360 instead

Reply:
Done. Thanks!